pkg/controller/cloud/nodecontroller.go

```diff
@@ -55,6 +65,13 @@ const (
 
 	// The amount of time the nodecontroller should sleep between retrying NodeStatus updates
 	retrySleepTime = 20 * time.Millisecond
+
+	//Taint denoting that a node needs to be processed by external cloudprovider
+	CloudTaintKey = "ExternalCloudProvider"
+
+	nodeStatusUpdateFrequency = 10 * time.Second
+
+	LabelProvidedIPAddr = "beta.kubernetes.io/provided-node-ip"
```

Comment:
I understand two nodes can't have the same IP at a given time but I'm wondering about the use case here, since I guess two nodes can have the same IP addresses at different times